src/integratedServer/worker.ts

```
@@ -0,0 +1,149 @@
+import { localFsState } from './browserfsShared'
+import { mountFsBackend, onWorldOpened } from './browserfsServer'
+
+let server: import('flying-squid/dist/index').FullServer & { options }
+
+export interface CustomAppSettings {
+  autoSave: boolean
+  stopLoad: boolean
+}
+
+export interface BackEvents {
+  ready: {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)